.env.example

```diff
@@ -1,12 +0,0 @@
-AWS_ACCESS_ID=
-AWS_SECRET_KEY=
-GMAPS_API_KEY=
-HOST_DOMAIN=
-MAILGUN_SMTP_LOGIN=
-MAILGUN_SMTP_PASSWORD=
-RECAPTCHA_SITE_KEY=
-RECAPTCHA_SECRET_KEY=
-S3_BUCKET=
-S3_REGION=
-STRIPE_PUBLISHABLE_KEY=
-STRIPE_SECRET_KEY=
```

Comment:
Presumably this was unintended, but can you restore this file?

Reply:
I can't restore it... I thought i'd created a .env, but now it isn't showing.

Reply:
The `.env` and `.env.example` are different files. You deleted all the stuff in `.env.example` -- either copy & paste it back, or `git checkout .env.example`, and it should come back.